adot-testbed/app/build.gradle.kts

```diff
@@ -0,0 +1,46 @@
+/*
+ * This file was generated by the Gradle 'init' task.
+ *
+ * This generated file contains a sample Java application project to get you started.
+ * For more details on building Java & JVM projects, please refer to https://docs.gradle.org/8.2/userguide/building_java_projects.html in the Gradle documentation.
+ */
+
+plugins {
+    // Apply the application plugin to add support for building a CLI application in Java.
+    application
```

Comment:
can you switch to type library so that we don't need to define an application section with a main class?

Reply:
Agreed